jtlibrary/python/jtlibrary/src/jtlib/segmentation.py

```diff
@@ -27,7 +27,6 @@ def detect_blobs(image, mask, threshold, min_area, deblend_nthresh=500,
         deblend_cont=0):
     '''Detects blobs in `image` using an implementation of
     `SExtractor <http://www.astromatic.net/software/sextractor>`_ [1].
-
```

Comment:
I'm not sure the documentation will still look correct if separation lines before the headings are removed.  Same comment applies on lines 47, 52, 106, 117, 185, 220, 237, and 241.